irc/pounce/Portfile

```diff
@@ -1,10 +1,15 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           legacysupport 1.1
+PortGroup           makefile 1.0
```

Comment:
the "makefile" PG is kind of intended for things that don't have a configure script. Why did add it here?

Reply:
@reneeotten [catgirl](https://github.com/macports/macports-ports/blob/master/net/catgirl/Portfile) uses this portgroup too (due to similar codebase), should I remove it there too?